mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping

```diff
@@ -2,9 +2,13 @@ CLASS dcp net/minecraft/client/network/ClientLoginNetworkHandler
 	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD b client Lcqn;
 	FIELD c parentGui Lcut;
+	FIELD d statusConsumer Ljava/util/function/Consumer;
 	FIELD e connection Liu;
 	FIELD f playerProfile Lcom/mojang/authlib/GameProfile;
 	METHOD <init> (Liu;Lcqn;Lcut;Ljava/util/function/Consumer;)V
 		ARG 1 connection
 		ARG 2 client
 		ARG 3 parentGui
+	METHOD a loginServer (Ljava/lang/String;)Ljg;
+		ARG 1 serverid
```

Comment:
`serverId`?

Reply:
Also, shouldn't the method be `loginToServer`/`login`/`onServerLogin`? The current name is implying that the server is the one being logged in.

Reply:
@gegy1000 This is based on AuthLib's Javadoc.

@Runemoro You're right.

Reply:
It's still incorrect to Java standard though, is it not?

Reply:
```     * @param serverId The random ID of the server to join```

Reply:
Oh, I'm referencing the capitalisation of the `i` in `id`

Reply:
Again, I'm sticking to the Javadocs here, which i think is *slightly* less confusing.

(Also,we tend to capitalize the **i** in our mappings)

Reply:
The mapping is `serverid`, it should be `serverId`

Reply:
Oh! I'm really not well today.

Reply:
Hehe, should've clarified better.